pkgs/tools/security/browserpass/default.nix

```diff
@@ -14,7 +21,7 @@ buildGoModule rec {
 
   vendorHash = "sha256-CjuH4ANP2bJDeA+o+1j+obbtk5/NVLet/OFS3Rms4r0=";
 
-  doCheck = false;
+  doCheck = stdenv.buildPlatform.canExecute stdenv.hostPlatform;
```

Comment:
This is the default, just set `doCheck = true`, and stdenv would take care of it.